modules/utils/monitor_phy_param.py

```diff
@@ -0,0 +1,52 @@
+import getopt
+import sys
+import datetime
+import subprocess
+from time import sleep
+from threading import Thread
+
+#defult rssi monitoring interval: 5sec
+rssi_mon_interval = 5
+#defualt interface
+interface = "wlan0"
+
+def get_rssi():
+    global interface
+    cmd = "iw dev " + interface + " station dump | grep 'signal:' | awk '{print $2}'"
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
+    rssi = proc.communicate()[0].decode('utf-8')
+    return rssi
+
+def log_rssi():
+    global rssi_mon_interval
+    fn_suffix=str(datetime.datetime.now())
+    log_file_path = '/var/log/'
+    log_file_name =  'rssi'+fn_suffix+'.txt'
+    while True:
+        f = open(log_file_path+log_file_name, 'a')
+        rssi_sta = get_rssi()
+        print(rssi_sta)
+        f.write(rssi_sta)
+        f.close()
+        sleep(rssi_mon_interval)
+
+if __name__=='__main__':
+    argv = sys.argv[1:]
+
+    try:
+        opts, args = getopt.getopt(argv, "hr:i:")
+        for opt, arg in opts:
```

Comment:
isn't it better to use arg.parser? 
https://docs.python.org/3/library/argparse.html

Reply:
Yes, this seems easy to use, will change to argparser.